packages/core/src/extensions/SideMenu/SideMenuPlugin.ts

```diff
@@ -511,6 +525,17 @@ export class SideMenuView<
     }
   };
 
+  update() {
```

Comment:
Let's add a comment why (including an example) why this is necessary, so we don't forget in the future